model/metrics/system-metrics.yaml

```diff
@@ -77,6 +77,14 @@ groups:
     unit: "By"
     attributes: []
 
+  - id: metric.system.memory.shared
+    type: metric
+    metric_name: system.memory.shared
+    stability: experimental
+    brief: "Reports shared memory. Memory used (mostly) by tmpfs."
```

Comment:
```suggestion
    brief: "Shared memory used as reported by the [`free` command](https://man7.org/linux/man-pages/man1/free.1.html)."
```

?

Reply:
Thank's. I have moved the report to the `free` command in the `note` section to avoid implying that the usage of this specific command is required. The same information can be collected directly from `/proc/meminfo`. Which is what the [Collector does](https://github.com/open-telemetry/semantic-conventions/issues/522#issuecomment-2047653901).